dev/mkmrconfig.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $MKMRCONFIG_TRACE ]] && set -x
+
+# Scans Github organizations and emits a config file for use with `myrepos`.
+#
+# Dependencies:
+#   myrepos -- https://myrepos.branchable.com
```

Comment:
It's the official project homepage and "Last edited 22 days and 4 hours ago" but yeah it's debatable whether that's the best entrypoint. Gonna add the github url and install url.